Ryujinx/Ui/GLScreen.cs

```
@@ -44,6 +44,114 @@ protected override void OnLoad(EventArgs e)
 
             Renderer.FrameBuffer.SetWindowSize(Width, Height);
         }
+        
+        private bool IsGamePadButtonPressedFromString(GamePadState GamePad, string Button)
+        {
+            if (Button == "LTrigger" || Button == "RTrigger")
+            {
+                return GetGamePadTriggerFromString(GamePad, Button) >= Config.GamePadTriggerThreshold;
+            }
+            else
+            {
+                return (GetGamePadButtonFromString(GamePad, Button) == ButtonState.Pressed);
+            }
+        }
+
+        private ButtonState GetGamePadButtonFromString(GamePadState GamePad, string Button) //Please make this prettier if you can.
```

This comment is not necessary...

---

I just though it looked kinda ugly, but okay I can remove it.

---

You could just return the enum value in each case directly. And then placing it all on a single line, like this:
```
case "A": return GamePad.Buttons.A;
```
Then you can throw an exception in the case the value is invalid and is not handled by the switch at the end of the method. This eliminates the need to print to the console (and for a default value).